nextflow.config

```diff
@@ -27,7 +27,7 @@ params {
     // Config options
     config_profile_name        = null
     config_profile_description = null
-    custom_config_version      = 'master'
+    custom_config_version      = 'main'
```

Comment:
This should actually remain as `master` since it's used as part of a URL in for nf-core configs.

Reply:
I got a little overzealous on my switching to `main` [4fe0406](https://github.com/phac-nml/staramrnf/pull/20/commits/4fe0406ae17756a5e925664194e4fa256c12fe4a) I have restored them

Reply:
3d6e79a